modules/Xsens_module/src/XsensRetargeting.cpp

```diff
@@ -3,7 +3,16 @@
 #include <iterator>
 #include <sstream>
 
-XsensRetargeting::XsensRetargeting(){};
+class XsensRetargeting::impl
+{
+public:
+    bool mapJointsHDE2CONTROLLER(std::vector<std::string> robotJointsListNames,
```

Comment:
Can you add the documentation related to this function? Furthermore is there a reason why `mapJointsHDE2CONTROLLER` is not called `mapJointsHDE2Controller`?

Reply:
updated.